plugins/nf-azure/src/main/nextflow/cloud/azure/batch/AzBatchService.groovy

```diff
@@ -585,10 +590,10 @@ class AzBatchService implements Closeable {
         final poolParams = new PoolAddParameter()
                 .withId(spec.poolId)
                 .withVirtualMachineConfiguration(poolVmConfig(spec.opts))
-                // https://docs.microsoft.com/en-us/azure/batch/batch-pool-vm-sizes
+        // https://docs.microsoft.com/en-us/azure/batch/batch-pool-vm-sizes
                 .withVmSize(spec.vmType.name)
-                // same as the num ofd cores
-                // https://docs.microsoft.com/en-us/azure/batch/batch-parallel-node-tasks
+        // same as the num ofd cores
+        // https://docs.microsoft.com/en-us/azure/batch/batch-parallel-node-tasks
                 .withTaskSlotsPerNode(spec.vmType.numberOfCores)
```

Comment:
think we still have problem tabs vs space. NF code uses 4 spaces for tab.

Reply:
whoops, that's IDEA going its own way! I didn't notice the reformatting, I'll fix it.

Reply:
I "should" be OK now.